recipes/blosc2/meta.yaml

```diff
@@ -0,0 +1,88 @@
+# Jinja variables
+{% set name = "blosc2" %}
+{% set version = "3.0.0rc3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Blosc/python-blosc2/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9186daeb69bdffefc222063e3cb7c01185b6f794b5fa42f4d190761a556fdd62
+
+build:
+  number: 0
+  skip: true  # [py<311]
+  script_env:
+    - USE_SYSTEM_BLOSC2=1
+  script: |
+    {% if win %}
+    git clone https://github.com/microsoft/vcpkg.git
+    .\vcpkg\bootstrap-vcpkg.bat
+    .\vcpkg\vcpkg install pkg-config
```

Comment:
in short, vcpkg is microsoft's answer to conda.

A different strategy. 

At conda-forge, you must build everything (well most things) with conda-forge's stack.